sql/sys_vars.cc

```diff
@@ -6773,15 +6773,20 @@ static Sys_var_mybool Sys_binlog_encryption(
        READ_ONLY GLOBAL_VAR(encrypt_binlog), CMD_LINE(OPT_ARG),
        DEFAULT(FALSE));
 
-static const char *binlog_row_image_names[]= {"MINIMAL", "NOBLOB", "FULL", NullS};
+static const char *binlog_row_image_names[]=
+{
+  "MINIMAL", "NOBLOB", "FULL", "FULL_DIFFERENT", NullS
+};
 static Sys_var_on_access<Sys_var_enum,
                          PRIV_SET_SYSTEM_VAR_BINLOG_ROW_IMAGE,
                          PRIV_SET_SYSTEM_VAR_BINLOG_ROW_IMAGE>
 Sys_binlog_row_image(
        "binlog_row_image",
-       "Controls whether rows should be logged in 'FULL', 'NOBLOB' or "
-       "'MINIMAL' formats. 'FULL', means that all columns in the before "
-       "and after image are logged. 'NOBLOB', means that mysqld avoids logging "
+       "Controls whether rows should be logged in 'FULL', 'FULL_DIFFERENT', "
+       "'NOBLOB' or 'MINIMAL' formats. 'FULL', means that all columns in the "
+       "before and after image are logged. 'FULL_DIFFERENT', means that all "
+       "columns are logged in before image, but only changed columns are "
```

Comment:
Considering [clarification](https://github.com/MariaDB/server/pull/2803/commits/300ead12bfa08dd15d1dd15dc201371e0e605ff6#r1399923916), let's specify about the inserted columns. ? How about 
```
'FULL_DIFFERENT', means that all columns are logged in before image, but only updated columns or all columns of the inserted record are logged  in before image,
```

Reply:
sure, it will update.